packages/ui/src/Slider/index.tsx

```diff
@@ -0,0 +1,213 @@
+import { theme } from '../PenumbraUIProvider/theme';
+
+interface SliderProps {
+  min?: number;
+  max?: number;
+  step?: number;
+  defaultValue?: number;
+  onChange?: (value: number) => void;
+  leftLabel?: string;
+  rightLabel?: string;
+  showValue?: boolean;
+  valueDetails?: string;
+  showTrackGaps?: boolean;
+  trackGapBackground?: string;
+  showFill?: boolean;
+  fontSize?: string;
+}
+
+const THUMB_SIZE = '16px';
+const TRACK_HEIGHT = '4px';
+const TRACK_GAP_WIDTH = '4px';
+
+const SliderContainer = styled.div`
+  position: relative;
+  width: 100%;
+  height: ${THUMB_SIZE};
+  cursor: pointer;
+  touch-action: none;
+`;
+
+const SliderTrack = styled.div`
+  position: absolute;
+  width: 100%;
+  top: 50%;
+  transform: translateY(-50%);
+  height: ${TRACK_HEIGHT};
+  background: rgba(250, 250, 250, 0.1);
+`;
+
+const SliderGap = styled.div<{ $left: number; $background: string }>`
+  position: absolute;
+  z-index: 2;
+  width: ${TRACK_GAP_WIDTH};
+  height: ${TRACK_HEIGHT};
+  background: ${props => props.$background};
+  left: ${props => props.$left}%;
+  transform: translateX(-50%);
+`;
+
+const SliderThumb = styled.div<{ $left: number }>`
+  position: absolute;
+  z-index: 3;
+  width: ${THUMB_SIZE};
+  height: ${THUMB_SIZE};
+  background: ${props => props.theme.color.neutral.contrast};
+  border-radius: 50%;
+  left: ${props => props.$left}%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  cursor: grab;
+  touch-action: none;
+`;
+
+const SliderFill = styled.div<{ percentage: number }>`
+  position: absolute;
+  z-index: 1;
+  height: ${TRACK_HEIGHT};
+  background: ${props => props.theme.color.primary.main};
+  border-radius: 2px;
+  top: 50%;
+  transform: translateY(-50%);
+  pointer-events: none;
+  width: ${props => props.percentage}%;
+`;
+
+const LabelContainer = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  margin-bottom: ${props => props.theme.spacing(1)};
+`;
+
+const Label = styled.div<{ $position: 'left' | 'right'; $fontSize: string }>`
+  ${detailTechnical}
+  font-size: ${props => props.$fontSize};
+  color: ${props => props.theme.color.text.secondary};
+  justify-self: ${props => (props.$position === 'left' ? 'flex-start' : 'flex-end')};
+`;
+
+const ValueContainer = styled.div<{ $fontSize: string }>`
+  ${detail}
+  display: flex;
+  margin-top: ${props => props.theme.spacing(2)};
+  border: 1px solid ${props => props.theme.color.other.tonalStroke};
+  font-size: ${props => props.$fontSize};
+  color: ${props => props.theme.color.text.primary};
+  padding: ${props => props.theme.spacing(2)} ${props => props.theme.spacing(3)};
+`;
+
+const ValueDisplay = styled.div`
+  color: ${props => props.theme.color.text.primary};
+`;
+
+const ValueDetails = styled.div`
+  margin-left: ${props => props.theme.spacing(1)};
+  color: ${props => props.theme.color.text.secondary};
+`;
+
+export const Slider: React.FC<SliderProps> = ({
```

Comment:
issue (blocking): this Slider is not a11y-compliant. You can check the [W3C standards for a Slider](https://www.w3.org/WAI/ARIA/apg/patterns/slider/) – it describes the patterns we should implement to provide the keyboard accessibility and help the users with low vision. For example, the standards say that the value can be controlled via pressing left or arrow keys when the Slider is focused.

So far, Jesse, Sam and me tried to comply with a11y standards in the design and code. If you open v2 pages, tabbing around shows correct focus state for any component, and it responds not only to the "enter press" but also to any other keyboard key. I believe we could do the same with Slider 🙏

suggestion: use [Radix's Slider component](https://www.radix-ui.com/primitives/docs/components/slider) as a base to handle the a11y and even simplify the code.